runbooks/deploy-to-multisite.md

```diff
@@ -0,0 +1,112 @@
+
+* Github JOTW Dev access
+* AWS access
+* CircleCi Access
+* You'll also need to be connected to the VPN.
+
+## Phases
+
+This assumes this is a code deployment for a theme (Hale) or plugin that the team supports/manages. If deploying an update to a third party theme/plugin skip to Phase III
+
+### Phase I - Create a new release of theme/plugin on github
+1. Create a new branch locally of the theme/plugin repo and commit the code changes to this branch. Make sure to update/bump the version number in plugin/theme files.
+
+2. Push the new branch up to github and raise a Pull Request to merge these changes into the main branch.
+
+3. Once the Pull Request has been approved merge it.
+
+4. Go to the releases section in theme/plugin git repo and draft a new release
+
+5. In the draft release form enter the following:
+- Tag version - this should be the latest version number of the theme/plugin (the same as updated in the files)
+- Release Title - a short title of what has been changed e.g name of new feature or Bug Fix. 
+- Release description - create a list of all the changes/ new features /bug fixes that will be released
+
+6. Publish the release
+
+### Phase II - Update Theme/Plugin Composer repo
+
+1. Login to [CirleCi](https://circleci.com/vcs-authorize/)
+
+2. Go to the CircleCi dashboard to view all pipelines. 
+
+3. Find the the pipeline `pp-satis-config` and rerun the workflow from the start using the action button. 
+
+4. Once this has complete (should only take a few minutes) the status should change from running to success 
+
+5. Check the latest release of the theme/plugin is in the [Composer repo](https://composer.wp.dsd.io/). You will need to be on the VPN to see this.
+
+
+### Phase III - Update multisite to bring in latest theme/plugin
+
+1. Make sure you are on the VPN
+
+1. Use the terminal tool and go to the root folder of the multisite. 
+
+2. Switch to master branch (if not already on) and git pull any latest changes down.
+
+3. Create and switch to new a branch from master to push the latest theme/plugin updates to.
+
+4. In terminal (at the root folder of the multisite). run `composer update`
+
+5. This will go through every package listed in the `composer.json` file and update where needed. Watch as it is doing this as other themes/plugins or even Wordpress may be updated.
+
+6. Run `git status`. The `composer.lock` should have been modified. 
+
+7. Spin up the multisite and test locally the latest changes and to make sure there are no issues from the update.
+
+8. Commit the changes to `composer.lock` to the update branch and push the branch to github
+
+9. Raise a Pull Request to merge the changes to master branch
+
+10. Once the Pull Request has been approved merge it. 
+
+11. This will now trigger a rebuild of the multisite on AWS
+
+
+### Phase IV - Test and Push the updates through the pipeline
+
+1. Login to the [AWS Console](https://aws.amazon.com/console/) 
+
+2. Go to CodePipeline section in AWS console.
```

Comment:
"Check you're in the right AWS region (Ireland). If not it can be switched in the to right corner drop down."